### DISCUSSION TITLE
音频独占相关的问题

### DISCUSSION THREAD

**Discussion by xcsnowcity** [hooke007/mpv_PlayKit#30] [Q&A 问答] [answered]
你好，我不知道这是不是bug.

复现如下：
mpv 设置为 WASAPI Exclusive,
用也设置为WASAPI Exclusive mode 的别的软件先播放视频/音频文件，这时再用mpv播放任何文件都会导致mpv崩溃。

而别的软件在音频通道被独占时，就算不能播放，也只是会弹出“音频通道无法播放”、“文件播放失败”之类的提示，而不是整个软件直接停止工作。

**Comment by hooke007** [comment, marked as answer]
我的建议是导出log然后反馈mpv官方。

我的经历独占冲突时只会卡顿。

**Reply by xcsnowcity** [reply]
👌，多谢回复，我去搞一搞。
